registry/storage/tombstone.go

```diff
@@ -0,0 +1,55 @@
+package storage
+
+// Functions for manipulating tombstones
+
+import (
+	"github.com/docker/distribution/context"
+	"github.com/docker/distribution/digest"
+	storagedriver "github.com/docker/distribution/registry/storage/driver"
```

Comment:
No need to keep the old package name for new files. The package used to be called `storagedriver` so we used this during the refactor. Please use `driver` directly here.